### PR TITLE
Refresh .NET article: Create windows service app

### PR DESCRIPTION
**Pull request by dksimpson**
## Summary
Task 1409933: https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1409933/

### CODE REVIEW

**Review by rpetrusha — CHANGES_REQUESTED** [review]
I've left a large number of comments and suggestions, @dksimpson, but this is a good revision of a very stale article. Thanks for doing it.

File: docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md [hunk 1]
```diff
@@ -10,50 +10,58 @@ helpviewer_keywords:
...
+1. In the **Service1.cs [Design]** tab for **Service1.cs** (or **Service1.vb [Design]** for **Service1.vb**), select the **switch to code view** link. 
 
-2. In the **Properties** window for **Service1.cs [Design]** or **Service1.vb [Design]**, change the **ServiceName** value to **MyNewService**.
+    The **Service1.cs** window appears.
 
-3. In **Solution Explorer**, rename **Service1.cs** to **MyNewService.cs**, or rename **Service1.vb** to **MyNewService.vb**.
+2. In the **Service1.cs** window, select any instance of `Service1` in the code, and then choose **Rename** from the shortcut menu. Replace `Service1` with *MyNewService* in the code and then either press **Enter** or select **Apply** in the **Rename: Service1** pop-up window.
+
+3. Select the **Service1.cs [Design]** (or **Service1.vb [Design]**) tab and select **Properties** from the shortcut menu. From the **Properties** window, change the **ServiceName** value to *MyNewService*.
```
[the remark this answers is not in the document]
Comment (dksimpson) [reply docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md]: You're correct - if the user renames the file first, VS will then prompt to rename the default Service1 references.

File: docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md [hunk 2]
```diff
@@ -10,50 +10,58 @@ helpviewer_keywords:
...
+1. In **Solution Explorer**, from the shortcut menu for **MyNewService.cs** (or **MyNewService.vb**), choose **View Designer**.
 
-2. From the **Components** section of the **Toolbox**, drag an <xref:System.Diagnostics.EventLog> component to the designer.
+2. In **Toolbox**, expand **Components**, and then drag the **EventLog** component to the **Service1.cs [Design]** (or **Service1.vb [Design]**) tab.
 
-3. In **Solution Explorer**, open the context menu for **MyNewService.cs** or **MyNewService.vb**, and then choose **View Code**.
+3. In **Solution Explorer**, from the shortcut menu for **MyNewService.cs** (or **MyNewService.vb**), choose **View Code**.
 
-4. Edit the constructor to define a custom event log:
+4. Edit the `MyNewService()` constructor to define a custom event log:
```
[the remark this answers is not in the document]
Comment (dksimpson) [reply docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md]: VS adds these System.Diagnostics by default in Service1.cs when you create a C# Windows service project.  However VS doesn't add any includes for a VB project.

**Comment by dksimpson** [comment]
Hello @rpetrusha, I've incorporated your updates. Let me know if there's anything else you'd like me to change.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thank you for making the additional changes, @dksimpson. I'll merge your PR now.

**Pull request merged**